src/Microsoft.Health.Dicom.Api/Modules/SecurityModule.cs

```diff
@@ -41,26 +47,60 @@ public void Load(IServiceCollection services)
 
         if (_securityConfiguration.Enabled)
         {
-            string[] validAudiences = GetValidAudiences();
-            string challengeAudience = validAudiences?.FirstOrDefault();
-
-            services.AddAuthentication(options =>
+            AuthenticationBuilder authenticationBuilder = services.AddAuthentication(options =>
             {
                 options.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
                 options.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
                 options.DefaultScheme = JwtBearerDefaults.AuthenticationScheme;
-            })
-            .AddJwtBearer(options =>
-            {
-                options.Authority = _securityConfiguration.Authentication.Authority;
-                options.RequireHttpsMetadata = true;
-                options.Challenge = $"Bearer authorization_uri=\"{_securityConfiguration.Authentication.Authority}\", resource_id=\"{challengeAudience}\", realm=\"{challengeAudience}\"";
-                options.TokenValidationParameters = new TokenValidationParameters
-                {
-                    ValidAudiences = validAudiences,
-                };
             });
 
+            AddAuthenticationSchemes(authenticationBuilder);
+
+            authenticationBuilder.AddPolicyScheme(
+                JwtBearerDefaults.AuthenticationScheme,
+                JwtBearerDefaults.AuthenticationScheme,
+                options =>
+                {
+                    options.ForwardDefaultSelector = context =>
+                    {
+                        // Find the first authentication header with a JWT Bearer token whose issuer
+                        // contains one of the scheme names and return the found scheme name.
+                        StringValues authHeaders = default(StringValues);
+
+                        if (!StringValues.IsNullOrEmpty(context.Request.Headers.Authorization))
+                        {
+                            authHeaders = context.Request.Headers.Authorization;
+                        }
+                        else if (!StringValues.IsNullOrEmpty(context.Request.Headers.WWWAuthenticate))
```

Comment:
I believe that `WWW-Authenticate` is a response header. I don't know of a case where we should do something with it on a request - https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/WWW-Authenticate

Reply:
The example I found online was also checking that header so I decided to mimic that - https://stackoverflow.com/questions/52952600/multiple-jwt-authorities-issuers-in-asp-net-core

But after a quick search it looks like you are right, its mainly a response header. I will remove this check